### PR TITLE
MCLOUD-6896: Add possibility to pass ECE-Tools environment variables

### PR DESCRIPTION
**Pull request by oshmyheliuk**
## Purpose of this pull request

This pull request (PR)  will add info on how to configure environment variables for docker containers.

## Affected DevDocs pages

-  https://devdocs.magento.com/cloud/docker/docker-config.html

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: src/cloud/docker/docker-config.md [hunk 1]
```diff
@@ -46,6 +46,31 @@ To skip the interactive mode, use the `-n, --no-interaction` option.
 {:.bs-callout-info}
 The mode option for the `ece-docker build:compose` command does not affect the Magento mode. It determines the {{site.data.var.ece}} file system installation and read-only or read-write behavior.
 
+## Set the environment variables
+
+You can launch a Docker with predefined environment variable by passing them in `--env-vars` option.
```
Comment (meker12) [inline src/cloud/docker/docker-config.md:51]:
```suggestion
You can launch a Cloud Docker environment with predefined environment variables by adding the `--env-vars` option to the `ece-docker build:compose` command.
```

File: src/cloud/docker/docker-config.md [hunk 2]
```diff
@@ -46,6 +46,31 @@ To skip the interactive mode, use the `-n, --no-interaction` option.
...
+
+For example,
+```
+bin/ece-docker build:compose --env-vars="{\"MAGENTO_CLOUD_VARIABLES\":{\"LOCK_PROVIDER\":\"db\",\"CRON_CONSUMERS_RUNNER\":{\"cron_run\":\"true\",\"max_messages\":5000,\"consumers\":[\"test\"]}}}"
+```
+
+Use the next php script to generate value for `--env-vars` option:
```
Comment (meker12) [inline src/cloud/docker/docker-config.md:58]:
```suggestion
You must escape special characters when specifying the value for the `--env-vars` option. Use the following PHP script to generate the escaped value for the configuration. Update the values with the values required for your Cloud Docker environment configuration.
```

**Review by meker12 — CHANGES_REQUESTED** [review]

File: src/cloud/docker/docker-config.md [hunk 3]
```diff
@@ -46,6 +46,31 @@ To skip the interactive mode, use the `-n, --no-interaction` option.
...
+
+You can launch a Cloud Docker environment with predefined environment variables by adding the `--env-vars` option to the `ece-docker build:compose` command.
+
+For example,
+```
+bin/ece-docker build:compose --env-vars="{\"MAGENTO_CLOUD_VARIABLES\":{\"LOCK_PROVIDER\":\"db\",\"CRON_CONSUMERS_RUNNER\":{\"cron_run\":\"true\",\"max_messages\":5000,\"consumers\":[\"test\"]}}}"
+```
+
+You must escape special characters when specifying the value for the `--env-vars` option. Use the following PHP script to generate the escaped value. Update the example with the values required for your Cloud Docker environment configuration.
+```php
```
Comment (meker12) [inline src/cloud/docker/docker-config.md:59]:
```suggestion

```php
```

**Review by meker12 — APPROVED** [review]

**Comment by meker12** [comment]
running tests

**Pull request merged**